server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/SystemInfoPage.java

```diff
@@ -30,17 +30,17 @@ public SystemInfoPage() {
   }
 
   public SystemInfoPage shouldHaveCard(String title) {
-    Selenide.$$(".system-info-health-card-title").find(Condition.text(title)).should(Condition.exist);
+    Selenide.$$(".boxed-group-accordion-title").find(Condition.text(title)).should(Condition.exist);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal ".boxed-group-accordion-title" 3 times. ([squid:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2FSONAR-10347_deliveries&issues=AWGJIXp0uvYJzwp2Y7wZ&open=AWGJIXp0uvYJzwp2Y7wZ)

[ISSUE_KEY]: AWGJIXp0uvYJzwp2Y7wZ